test/SecTester.Scan.Tests/Models/ModelEnumJsonSerializationTests.cs

```diff
@@ -0,0 +1,152 @@
+using SecTester.Bus.Dispatchers;
+using Module = SecTester.Scan.Models.Module;
+
+namespace SecTester.Scan.Tests.Models;
+
+public class JsonStringEnumMemberConverterTests
+{
+  private readonly DefaultMessageSerializer _sut = new();
+
+  public static readonly IEnumerable<object[]> AttackParamLocationEnumerable = new List<object[]>()
```

Comment:
The same for all of them below

```suggestion
  public static readonly IEnumerable<object[]> AttackParamLocationEnumerable = new List<object[]>
```